src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -300,51 +313,51 @@ Content-Type: application/xml
+* Some elements, such as `AirlineTickets` or `RailPayments`, appear only for bookings of the appropriate type. For example `AirlineTickets` appears in the response only for air bookings and `RailPayments`, for rail bookings.
 * Amount values, such as Rate or Tax, appear only if the requestor is the source of the booking. All other suppliers will not receive the amount elements associated with the bookings.
-* Some elements, such as SabreDKNumber, appear only if the booking was created by the relevant GDS.
+* Some elements, such as `SabreDKNumber`, appear only if the booking was created by the relevant GDS.
 * Some elements are vendor-specific and appear only in responses for the associated vendor.  
 
 This topic describes the full set of possible elements that can be returned. No itinerary can contain all of the possible elements, so the response will always be a subset of all the possible returned values.    
 
-By default, when calling this API, the Concur account associated with the OAuth access token used to make the API call should be the owner of the trip. This endpoint can also be used to get details for trips that the OAuth consumer does not own. This is most often done when a Travel Management Company needs to get trip details on behalf of a user. The TMC must be registered with Concur and have a Concur account that has one of the following user roles: Web Services Administrator for Professional, or Can Administer for Standard.
+By default, when calling this API, the SAP Concur account associated with the OAuth access token used to make the API call should be the owner of the trip. This endpoint can also be used to get details for trips that the OAuth consumer does not own. This is most often done when a TMC needs to get trip details on behalf of a user. The TMC must be registered with SAP Concur and have an SAP Concur account that has one of the following user roles: Web Services Administrator for Professional, or Can Administer for Standard.
 
 ## Request
 
-  GET /travel/trip/v1.1/trip_ID?[systemFormat=system_format|&userid_type=login|&user_id=login_ID]
+```GET /travel/trip/v1.1/trip_ID?[systemFormat=system_format|&userid_type=login|&user_id=login_ID]```
 
 ## Path Parameters
 
 | Parameter Name | Data Type |Description
 | --------- | --------- | -------
-| trip_ID (required) | string  |  The identifier for the desired trip. This identifier is returned as the value of the id element when getting trip summaries. For example, if the returned value of the id element is I2uwiJJw8r7Owl3IWlSie9WIelxhAhwiL, then the URI for the request is `/travel/trip/v1.1/I2uwiJJw8r7Owl3IWlSie9WIelxhAhwi`
+| trip_ID (required) | string  |  The identifier for the desired trip. This identifier is returned as the value of the ID element when getting trip summaries. For example, if the returned value of the ID element is I2uwiJJw8r7Owl3IWlSie9WIelxhAhwiL, then the URI for the request is `/travel/trip/v1.1/I2uwiJJw8r7Owl3IWlSie9WIelxhAhwi`.
```

Comment:
Let's use the standard bold Required in the description

Reply:
Fixed, note there is a table that has a column for optional v. required and I left that as is.